### PR TITLE
Migrate Easy Dune Queries

### PR DESCRIPTION
**Pull request by bh2smith**
Just a few queries that have been converted to DuneSQL.

### CODE REVIEW

File: queries/dune_v2/vouch_registry.sql [hunk 1]
```diff
@@ -1,14 +1,14 @@
 -- -- V2 Query: https://dune.com/queries/1541516
```
Comment (gentrexha) [inline queries/dune_v2/vouch_registry.sql]: Nit: Update comment to reflect V3

File: src/fetch/dune.py [hunk 2]
```diff
@@ -163,24 +162,6 @@ def get_vouches(self) -> list[DuneRecord]:
             )
         )
 
-    def get_period_totals(self) -> PeriodTotals:
```
Comment (gentrexha) [inline src/fetch/dune.py:166]: Out of curiosity: What's the reason we removed this?

**Review by gentrexha — APPROVED** [review]
LGTM.

I would recommend updating the directory name from `dune_v2` to `dune_v3` as well.

**Comment by bh2smith** [comment]
> I would recommend updating the directory name from `dune_v2` to `dune_v3` as well.

Actually, I will be deleting the dune_v1 directory in a follow up PR and calling this directory only Dune.

File: src/fetch/dune.py (hunk 2 above)
Comment (bh2smith) [reply src/fetch/dune.py:166]: It is not used for anything (i.e. dead code).

**Pull request merged**